tests/test_sessions.py

```diff
@@ -0,0 +1,75 @@
+import os
+import tempfile
+import pytest
+import json
+from flasksrc import create_app, db
+from flask_jwt_extended import create_access_token
+
+@pytest.fixture
+def client():
+    app = create_app({'TESTING': True})
+    db_fd, app.config['DATABASE'] = tempfile.mkstemp()
+
+    with app.test_client() as client:
+        with app.app_context():
+            db.init_db()
+        yield client
+
+    os.close(db_fd)
+    os.unlink(app.config['DATABASE'])
+def test_topo_id_post(client):
+    """Check the POST topo_id returns a session id if valid or error if not"""
+    client.post(
+          '/api/user',
+          data={
+              'username': 'Arthur@projectreclass.org',
+              'password': 'BaLtH@$0R',
+              'first_name': 'Arthur',
+          },
+    )
+    url = "/api/toynet/session"
+
+    pv = ({'toynet_topo_id': 1, 'toynet_user_id': 'Arthur@projectreclass.org',})
+    rv = client.post(url, data = pv,) 
+
+    rv_json = json.loads(rv.data.decode('utf-8'))
+    #print(rv_json['message'])
+    assert rv.status_code == 201
+    assert rv_json['status'] == True
+    # check that a session id was returned
+    assert type(rv_json['toynet_session_id']) == int
+
+    #Invalid toynet_topo_id number
+    pv = ({'toynet_topo_id': -5, 'toynet_user_id': 'Arthur@projectreclass.org',})
```

Comment:
here as well, please apply this style to all test files, or make this test file match the rest :)

Reply:
`Arthur` -> `arthur`

Reply:
Done